docs/contributing/release_guide.rst

```diff
@@ -26,42 +19,27 @@ Throughout this guide, ``0.9.0`` denotes the version you're releasing.
 .. When updating this guide, make sure that each bullet point is for
    doing exactly one task!
 
-Several weeks before release
-----------------------------
+Announce timeline
+=================
 
 * Create an issue on GitHub for the release with a checklist of tasks
   to be performed.
 
-.. Should we create an issue template for this?  Should we convert the
-   checklist into an issue format?
+.. todo::
 
-* Three weeks before the release, announce that a feature freeze will
-  occur one week before the anticipated release date. Only pull requests
-  with a limited scope that do not significantly change functionality
-  should be merged during the feature freeze. |exclude bugfix|
+   Convert the release guide into an issue template with a checklist?
 
-* Announce a code freeze beginning ∼2–3 weekdays before the release.
-  Only bug fixes and pull requests directly related to the release
-  should be merged during the code freeze. |exclude bugfix|
+* Three weeks before a minor or major release, announce that a feature
+  freeze will occur one week before the anticipated release date. Only
+  pull requests with a limited scope that do not significantly change
+  functionality should be merged during the feature freeze.
 
-Pre-release
------------
+* Three weekdays before a minor or major release, announce a code
+  freeze. Only bugfixes and pull requests that are directly related to
+  the release should be merged during the code freeze.
```

Comment:
Not to hold up the PR, but this still feels like something we could avoid with sufficiently advanced ~technology~ ~magic~ Git usage.

Reply:
I'm all for automating as much of the release process that we reasonably can!